Common/POpen.cpp

```diff
@@ -109,6 +113,8 @@ POpen::POpen(const std::string& cmd) : m_Command(cmd)
 		break;
 	}
 
+	WaitForSingleObject(m_ProcInfo.hProcess, INFINITE);
```

Comment:
and here, we are waiting that the process stops?
imo one of the two is wrong

Reply:
ok, as discussed, we now have a better overview;
this "platform specific" code should be moved in a new POpen::WaitForMFA() function

Reply:
Applied deeper refactor creating P4MFA and the WaitForHelixMFA method